kubernetes/zenko/charts/backbeat/values.yaml

```diff
@@ -29,8 +29,7 @@ redis:
 health:
   port: 4042
   path:
-    liveness: /_/health/liveness
-    readiness: /_/health/readiness
+    liveness: /_/health/readiness
```

Comment:
I feel like there's a mismatch here

Reply:
@JianqinWang I will invite you to read the description 😅

Reply:
I see that, let me rephrase - as a whole I feel like this naming schema creates a naming inconsistency. However, if you feel this is the best way to resolve this and this kind of naming change can't be avoided, then I will approve.

Reply:
It's clearly not the best way. But it's the only way of solving this issue without having to rebuild the `Backbeat` image (which we only do for releases). The reason for expediting this change without an image rebuild is because this is critical (IMHO).

Reply:
Thanks for the explanation